test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/PushCommandTest.cs

```diff
@@ -444,31 +461,35 @@ public void PushCommand_Server_Nupkg_ByFilename_SnupkgDoesNotExist_NoFileNotFoun
         /// <summary>
         /// When pushing *.nupkg to a symbol server, but no snupkgs are selected with that wildcard, there is not a FileNotFound error about snupkgs.
         /// </summary>
-        [Fact]
-        public void PushCommand_Server_Nupkg_ByWildcard_SnupkgDoesNotExist_NoFileNotFoundError()
+        [Theory]
+        [InlineData(1)]
+        [InlineData(2)]
+        public void PushCommand_Server_Nupkg_ByWildcard_SnupkgDoesNotExist_NoFileNotFoundError(int numberOfPaths)
```

Comment:
@donnie-msft Is it now what you had in mind ?

Reply:
Nearly, yes. Since we do our own globbing, I'd still like to test `*.nupkg`. So rather than replace *.nupkg`, I'd like to ensure all 3 situations behave the same: 1 path, 2 paths, 1 wildcard path.

Reply:
OK, I've updated one test more. So now we test `1 path`, `2 paths`, `1 wildcard path` and `2 wildcard paths`.